incubator/hnc/internal/reconcilers/suite_test.go

```diff
@@ -66,7 +66,7 @@ var _ = BeforeSuite(func(done Done) {
 
 	By("bootstrapping test environment")
 	testEnv = &envtest.Environment{
-		CRDDirectoryPaths: []string{filepath.Join("..", "..", "config", "crd", "bases")},
+		CRDDirectoryPaths: []string{filepath.Join("..", "..", "manifests", "envtest")},
```

Comment:
Maybe worth a comment here about what it _should_ be after we fix the bug and why it's set to the value that it is, a link to the bug, etc etc